### ISSUE TITLE
Multiple RTU connection to clients

### ISSUE THREAD

**Issue by NintendoPlayer** [rossmann-engineering/EasyModbusTCP.NET#60]
Hi!

I don't really know if it's possible, but I would like to ask if, right now, I can connect to more than one modbus rtu client.  I want to know if my app can read addresses from two different clients on the same COM port and with the same BaudRate.

Would creating two different ModbusClient instances work?

Thank you!

**Comment by Padanian** [comment]
If by client you mean Master, in any modbusRTU network there can be only one Modbus Master and as many as 249 Modbus Slaves, provided that recent politically-correct lunacy requires the word "Master" to be replaced by "Primary" and "Slave" to be replaced by "Secondary"

**Comment by NintendoPlayer** [comment]
Yes, I'm also very confused by all this word-change madness.  I understand it, but I think it's confusing.  Anyway, I was refering to connect to more than one slave.

**Comment by Padanian** [comment]
Of course you can connect up to 249 slaves to one master, provided each of them have unique slave address.
